### ISSUE TITLE
Get twitter data

### ISSUE THREAD

**Issue by richi0** [TomiiPomii/Social-Media-Summary#5]
Write the twitter_data function [here](https://github.com/TomiiPomii/Social-Media-Summary/blob/development/django_data/summary/collectors/collect.py) to collect all the data defined in the SocialMediaUser class. This data is then rendered in the [result](https://github.com/TomiiPomii/Social-Media-Summary/blob/development/django_data/summary/templates/summary/result.html) template.

**Comment by richi0** [comment]
I start by integrating access to the twitter api.

**Comment by richi0** [comment]
I integrated the access to the twitter api and made some simple tests. To use it add these 4 enviroment variables:

```
TWITTER_CONSUMER_KEY=<YOUR KEY>
TWITTER_CONSUMER_SECRET=<YOUR SECRET>
TWITTER_ACCESS_TOKEN_KEY=<YOUR KEY>
TWITTER_ACCESS_TOKEN_SECRET=<YOUR SECRET>
```

I added the python-twitter package. To install it run `docker-compose build`  in the root directory

It is now possible to get data about specific twitter user. To test this function easily I added a management command. The management command runs the twitter_data function. Run the following command to test it:

`docker-compose exec web python manage.py get_twitter_data`
